### PR TITLE
Dash, add a column with the number of opened issues per app

### PR DESCRIPTION
**Pull request by nathanael-h**
Dash, add a column with the number of opened issues per app 

- [X] issues number if > 0
- [X] number with link to issues page in Gihub
- [X] sort by issues
- [X] filter by issues nbr > 0
- [X] Update translation files (Translate new strings to FR will be done with weblate after)

### CODE REVIEW

**Review by OniriCorpe — APPROVED** [review]

**Comment by nathanael-h** [comment]
I'll go on tomorrow with the todo list above, time to :zzz:

**Comment by nathanael-h** [comment]
New column for issues, with filter on apps with 1 or more issues. Work in progress. 
image

**Comment by nathanael-h** [comment]
Sort by issues number added : 
image

**Comment by Salamandar** [comment]
Really nice, thanks for the work. What's left to do ? Translations ?

File: store/templates/dash.html [hunk 1]
```diff
@@ -43,6 +44,8 @@ <h1 class="text-2xl font-bold">
         <option {% if request.args.get("sort") in [None, "stars"] %}selected{% endif %} value="stars">{{ _("Popularity stars") }}</option>
         <option {% if request.args.get("sort") == "main_branch_update" %}selected{% endif %} value="main_branch_update">{{ _("Last update on main/master branch") }}</option>
         <option {% if request.args.get("sort") == "testing_branch_update" %}selected{% endif %} value="testing_branch_update">{{ _("Last update on testing branch") }}</option>
+        <option {% if request.args.get("sort") == "nb_issues" %}selected{% endif %} value="nb_issues">{{ _("Issues number") }}</option>
```
Comment (alexAubin) [inline store/templates/dash.html:47]:
```suggestion
        <option {% if request.args.get("sort") == "nb_issues" %}selected{% endif %} value="nb_issues">{{ _("Number of opened issues") }}</option>
```

**Comment by nathanael-h** [comment]
Yes, the only thing left is translations. But I don't get it. Does the store translated using weblate or not?

**Comment by alexAubin** [comment]
Yes it's done through weblate, but you may need to at least regenerate the base .po file I think ? Weblate won't run Flask-Babel itself

(instructions are in the README :wink: )

**Comment by nathanael-h** [comment]
Yep, is enough like this? Or is `pybabel update -i messages.pot -d translations && pybabel compile -d translations` also needed? Sorry, even with the README I don't get what should be done manually and what is automatically handled by weblate.

**Comment by alexAubin** [comment]
I think you need this too yes ... as far as I understand, weblate will just work with the raw .po file (and maybe regenerate the .mo from the .po everytime they get updated by translators) (It's not 100% clear to me either)

**Comment by nathanael-h** [comment]
Done, the PR is ready for review.

**Pull request merged**